File: debugModule/src/main/libraryKotlin/java/com/plugin/debug/librarykotlin/LibraryKotlinMainActivity.java
package com.plugin.debug.librarykotlin;

import android.os.Bundle;
import android.widget.Toast;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import com.plugin.debug.R;

public class LibraryKotlinMainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity_layout);
        Toast.makeText(this,"LibraryKotlinMainActivity",Toast.LENGTH_SHORT).show();
//        setContentView(R.layout.main_activity_layout);
//        ComponentManager.init(getApplication());
//        ((TextView) findViewById(R.id.text)).setText(SdkManager.getSdk(IProvideFromLibrary.class).provideString());
    }
}

File: libraryKotlin2/src/main/debug/java/com/plugin/librarykotlin2/LibraryKotlinMainActivity.kt
package com.plugin.librarykotlin2

import android.os.Bundle
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.plugin.component.ComponentManager
import com.plugin.component.SdkManager

//import com.plugin.library.ISdk

class LibraryKotlinMainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity_layout)
        ComponentManager.init(application)
        (findViewById<TextView>(R.id.text)).text = SdkManager.getSdk(IProvideFromLibraryKotlin2::class.java)!!.provideString()
    }
}
